feature-sessions/src/main/java/io/github/droidkaigi/confsched2022/feature/sessions/SessionDetail.kt

```diff
@@ -176,14 +183,32 @@ fun SessionDetailDescription(
     modifier: Modifier = Modifier,
     description: String,
 ) {
+    val (isReadMore, setIsReadMore) = remember { mutableStateOf(false) }
+    val (isOverFlow, setIsOverFlow) = remember { mutableStateOf(false) }
```

Comment:
Can you use Kotlin Delegated property?

`val (isReadMore, setIsReadMore) = remember { mutableStateOf(false) }`

↓

`val readmore by remember { mutableStateOf(false) }`

Because it causes recomposition to wider unneeded scope.

Reply:
I've corrected it based on your suggestion. Please check again. 🙏